packages/terra-framework-docs/src/terra-dev-site/doc/tabs/example/IconsInMenuTabs.jsx

```diff
@@ -55,7 +55,7 @@ const IconsOnlyAndInMenuTabs = (props) => {
 
   return (
     <div className={cx('content-wrapper')}>
-      <Tabs responsiveTo={props.responsiveTo}>
+      <Tabs responsiveTo={props.responsiveTo} id="icons-only">
```

Comment:
will the id be required prop ?

Reply:
It's optional

Reply:
responsiveTo is removed from Tabs !! do we still need this prop on example

Reply:
Removed in https://github.com/cerner/terra-framework/pull/1693/commits/793a2836d21b1f2e39b9af4241fa56e209b90a20